docs/builds/guides/frameworks/angular.md

```diff
@@ -271,6 +311,8 @@ The following `@Output` properties are supported by the CKEditor 5 rich text edi
 Fired when the editor is ready. It corresponds with the [`editor#ready`](https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editor-Editor.html#event-ready) event.
 It is fired with the editor instance.
 
+Note that this method might be called multiple times. It is called whenever the editor crashes and is restarted, hence the editor property should not be kept internally (because it might change).
```

Comment:
```suggestion
Note that this method might be called multiple times. Apart from initialization, it is also called whenever the editor is restarted after a crash. Do not keep the reference to the editor instance internally, because it will change in case of restart. Instead, you should use `watchdog.editor` property.
```